trinity/console.py

```diff
@@ -56,9 +57,17 @@ def console(ipc_path: Path,
     Method that starts the chain, setups the trinity CLI and register the
     cleanup function.
     """
+    logger = logging.getLogger('trinity')
     if env is None:
         env = {}
 
+    # if ipc_path is not found, print a useful error message
+    if not Path(ipc_path).exists():
```

Comment:
`ipc_path` is already a `Path` object so you can do `ipc_path.exists()` directly.